### PR TITLE
Fixed the navbar issue

### LINKED ISSUE #63: Footer when it is in big screen
### Description

There is a problem with the footer.
When we are on a big screen, The footer must be at the bottom of the page.

### Screenshots

image

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by alisamadiii**
## Short Description
The things that We've brought changes to the website:

- Fixed the navbar issue to be at the bottom in every situation
- Added `id` for each contributor in JSON file.
- Used id for key values for each card
- Developed some parts of contributors' cards 

## Fixes Issue

Closes #63 

## Screenshots
image

### CODE REVIEW

File: components/Footer/Footer.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ import Logo from "../../public/Header img.png";
 const Footer = () => {
   // const bgcolor= "#DADADA"
   return (
-    <footer className=" w-full bg-[#121B24] dark:bg-[#DADADA] min-h-[200px] relative">
+    <footer className="absolute bottom-0 left-0 w-full bg-[#121B24] dark:bg-[#DADADA] min-h-[200px]">
```
Comment (jaynarayan-vaishnav) [inline components/Footer/Footer.js:11]: @AliReza1083  instead of absolute we can use the sticky display to make it stay at the bottom.

**Review by jaynarayan-vaishnav — APPROVED** [review]
Everything else LGTM 🙌🏻. Great Work on refactoring the code Ali.

File: components/Footer/Footer.js (hunk 1 above)
Comment (alisamadiii) [reply components/Footer/Footer.js:11]: Position sticky won't fix our problem.

We'll use absolute, and we've set the body to be relative, Whenever the body is large, the footer stick with Body.

Comment (jaynarayan-vaishnav) [reply components/Footer/Footer.js:11]: Got it 👍🏻 thanks for clarifying this 🙂

**Comment by jaynarayan-vaishnav** [comment]
Merging it now @AliReza1083

**Pull request merged**